### PR TITLE
Updated metric_type schema to support custom metrics

### PR DESCRIPTION
**Pull request by rohitsharma04**
- metric_type can be alphanumeric characters and it can also have
underscores

### CODE REVIEW

File: servicebroker/config/catalog.schema.json [hunk 1]
```diff
@@ -201,13 +201,8 @@
...
                 "metric_type": {
-                  "enum": [
-                    "memoryused",
-                    "memoryutil",
-                    "responsetime",
-                    "throughput"
-                  ],
-                  "type": "string"
+                  "type": "string",
+                  "pattern": "^[a-zA-Z0-9_]+$"
```
Comment (qibobo) [inline servicebroker/config/catalog.schema.json]: the metric type can start with a number? such as "999metric".

Comment (rohitsharma04) [reply servicebroker/config/catalog.schema.json]: @pradyutsarma @paltanmoy @boyang9527 @cdlliuy 
should custom metric name be allowed to start with number ?

Comment (boyang9527) [reply servicebroker/config/catalog.schema.json]: I think that is fine

**Comment by boyang9527** [comment]
LGTM

**Pull request merged**